.pre-commit-config.yaml

```diff
@@ -41,10 +41,11 @@ repos:
       - id: rst-directive-colons
       - id: rst-inline-touching-normal
       - id: text-unicode-replacement-char
-  # - repo: https://github.com/pre-commit/mirrors-mypy
-  #   rev: v1.6.0
-  #   hooks:
-  #     - id: mypy
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.6.0
+    hooks:
+       - id: mypy
+         exclude: "tests/input/"
```

Comment:
Isn't that on pyproject.toml? `mypy` should read it? 😕

Reply:
Might need a little help to find it then.

Reply:
I will check it, but later 😅

Just leave me a `# ToDo: Move to pyproject.toml` if you work on it and you fix the CI

Reply:
```suggestion
         # ToDo: Move to pyproject.toml
         exclude: "tests/input/"
```

Reply:
Actually, you are right 😅 

https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-exclude

> Note that this flag only affects recursive directory tree discovery, that is, when mypy is discovering files within a directory tree or submodules of a package to check. If you pass a file or module explicitly it will still be checked. 

So we need to explicitly "hide those files" from mypy 😕